Makefile

```diff
@@ -17,6 +17,13 @@ $(THRIFT_BUILDDIR)/baseplate/thrift/baseplate.thrift_buildstamp: baseplate/thrif
 	cp -r $(THRIFT_BUILDDIR)/$</baseplate/thrift/ baseplate/
 	touch $@
 
+$(THRIFT_BUILDDIR)/tests/integration/test.thrift_buildstamp: tests/integration/test.thrift
+	@echo SPECIAL $< $@
```

Comment:
Is some of this a makefile specific thing, or do i just know even less bash than I thought I did? (particularly `@echo`)